Tools/TestWebKitAPI/Tests/WebKitCocoa/WritingTools.mm

```diff
@@ -3853,16 +3853,11 @@ InstanceMethodSwizzler swizzler(PAL::getWTWritingToolsClass(), @selector(showToo
     TestWebKitAPI::Util::run(&finished);
 }
 
-// FIXME rdar://148280675
-#if PLATFORM(IOS) && !defined(NDEBUG)
-TEST(WritingTools, DISABLED_IntelligenceTextEffectCoordinatorDelegate_RectsForProofreadingSuggestionsInRange)
-#else
 TEST(WritingTools, IntelligenceTextEffectCoordinatorDelegate_RectsForProofreadingSuggestionsInRange)
-#endif
 {
     RetainPtr session = adoptNS([[WTSession alloc] initWithType:WTSessionTypeProofreading textViewDelegate:nil]);
 
-    RetainPtr webView = adoptNS([[WritingToolsWKWebView alloc] initWithHTMLString:@"<body contenteditable><p id='first'>I don't thin so. I didn't quite here him.</p><p id='second'>Who's over they're. I could come their.</p></body>"]);
+    RetainPtr webView = adoptNS([[WritingToolsWKWebView alloc] initWithHTMLString:@"<body style='font-size: 14px; font-family: serif; line-height: 32px;' contenteditable><p id='first'>I don't thin so. I didn't quite here him.</p><p id='second'>Who's over they're. I could come their.</p></body>"]);
```

Comment:
May I recommend Ahem?

Reply:
Ah, didn't know that was a thing. Though it looks like it's not that trivial to use it in an API test looking at other tests that use it, so I think I prefer just doing it this way if that's ok